sys/usb/usbus/cdc/acm/cdc_acm_stdio.c

```diff
@@ -52,9 +52,12 @@ ssize_t stdio_read(void* buffer, size_t len)
 
 ssize_t stdio_write(const void* buffer, size_t len)
 {
-    usbus_cdc_acm_submit(&cdcacm, buffer, len);
-    usbus_cdc_acm_flush(&cdcacm);
-    /* Use tsrb and flush */
+    size_t n = 0;
+    do {
+        n += usbus_cdc_acm_submit(&cdcacm, buffer+n, len-n);
```

Comment:
I think you can just do

```suggestion
        n = usbus_cdc_acm_submit(&cdcacm, buffer, len);
        buffer += n;
        len -= n;
        …
    } while (len);
```

Reply:
Sure, but that way you lose how many chars were actually written out - which you should return from the function.

Reply:
Oh I see - but it will now always be `len`.
I just think it looks cleaner without the added calculations.

How about 
```C
const void* start = buffer;
…
return buffer - start;
```

Reply:
Agreed, looks cleaner. Just committed.

Reply:
Travis has some more suggestions:

> sys/usb/usbus/cdc/acm/cdc_acm_stdio.c:56: style (variableScope): The scope of the variable 'n' can be reduced.

>	sys/usb/usbus/cdc/acm/cdc_acm_stdio.c:61: portability (arithOperationsOnVoidPointer): 'buffer' is of type 'const void *'. When using void pointers in calculations, the behaviour is undefined.

casting them both to (char*) for the calculation at the end should be enough.